app/controllers/issues_controller.rb

```diff
@@ -2,18 +2,19 @@ class IssuesController < AuthenticatedController
   include ActivityTracking
   include ContentFromTemplate
   include ConflictResolver
-  include Mentions
+  include Mentioned
   include MultipleDestroy
   include NotificationsReader
   include ProjectScoped
 
   before_action :find_issuelib
   before_action :find_issues, except: [:destroy]
 
-  before_action :find_mentionable_users, only: [:show]
   before_action :find_or_initialize_issue, except: [:import, :index]
   before_action :find_or_initialize_tags, except: [:destroy]
 
+  include Subscribed
```

Comment:
really? down here? can't we insert the filter in the right spot? If not, we need a comment, otherwise someone refactoring this code will move this line with the other includes.